applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.hpp

```diff
@@ -0,0 +1,85 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:     BSD License
+//           license: structural_mechanics_application/license.txt
+//
+//  Main authors: Klaus B. Sautter
+//                   
+//                   
+//
+
+#if !defined(KRATOS_CR_BEAM_ELEMENT_LINEAR_3D2N_H_INCLUDED )
+#define  KRATOS_CR_BEAM_ELEMENT_LINEAR_3D2N_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_elements/cr_beam_element_3D2N.hpp"
+#include "includes/element.h"
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/serializer.h"
+
+namespace Kratos
+{
+
+	class CrBeamElementLinear3D2N : public CrBeamElement3D2N

```

Comment:
Ok, I see now there is a base class, Ok, could be something more general for beams and trusses?